### PR TITLE
Updates for JL 1.0.0-alpha.6

### PR DESCRIPTION
**Pull request by athornton**
Apologies for the editor-induced whitespace change.

### CODE REVIEW

**Comment by philippjfr** [comment]
Thanks much appreciated. Do you recommend merging and publishing a build on npm asap or holding off until beta, rc or even a real release is out?

**Comment by athornton** [comment]
Difficult to say.  I made the PR because I needed some of the new functionality in other extensions, which are targeting the 1.0.0-alpha series  (it's been a long time since 0.34).  I think I would suggest pushing this to npm (with an alpha version tag) but tagging it "next" rather than "latest".  We shouldn't do a new "latest" release until JL 1.0.0, I think.

**Comment by philippjfr** [comment]
Thanks, that's helpful, I've not looked into the distinction between next and latest releases but will follow your advice.

**Comment by stonebig** [comment]
as long as there is a no jupyterlab_pyviz for Jupyterlab-1.0.0a3 , so I can't use fresh altair-3.0.0 as it needs Jupyterlab-1.

**Comment by athornton** [comment]
@stonebig well, it'd be great if you tested against this one, because although it works for me, it'd be nice to have more data.

Here are the lines from my `Dockerfile` in which I install this so that it runs not-in-dev-mode.  I'm mixing it with a bunch more upstream fixes, but if you change `gitxt` to be just `pyviz_comms` you should be OK.  This installs by downloading into /usr/share/git and uses the system-installed Python 3.  `LBXT` in the last bit contains all my other lab extensions that I installed from npm.  For me that is: 

```
ENV LBXT="@jupyterlab/celltags @jupyterlab/toc \
          @jupyter-widgets/jupyterlab-manager \
          bqplot ipyevents ipyvolume jupyter-threejs \
          dask-labextension@0.4.0-pre \
          jupyter_firefly_extensions"
```

Which I install with:

```
RUN  set -e && \
      for l in ${LBXT}; do \
          jupyter labextension install ${l} --no-build ; \
      done
```
And then I install a bunch of extensions from git, which looks like this:

```
RUN  gitxt="jupyterlab-savequit jupyterlab-hub jupyterlab-lsstquery \
      pyviz_comms jupyterlab_bokeh" && \
      t="tickets/DM-18415" && \
      ls="https://github.com/lsst-sqre" && \
      cd /usr/share/git && \
      git clone -b ${t} ${ls}/pyviz_comms.git && \
      git clone -b ${t} ${ls}/jupyterlab-savequit.git && \
      git clone -b ${t} ${ls}/jupyterlab-lsstquery.git && \
      git clone -b ${t} ${ls}/jupyterlab-hub.git && \
      git clone -b ${t} ${ls}/jupyterlab_bokeh.git && \
      set -e && \
      for d in ${gitxt}; do \
          cd ${d} ; \
          jlpm install --unsafe-perm ; \
          jlpm run build ; \
          jupyter labextension install --no-build ; \
          cd .. ; \
      done
```
followed by (to enable all the extensions, both from npm and from git):
```
RUN  gitxt="pyviz_comms ipyaladin jupyterlab-savequit jupyterlab-hub \
      jupyterlab-lsstquery jupyterlab-server-proxy ndbime" && \
      set -e && \
      for l in ${LBXT} ${gitxt} ; do \
          jupyter labextension enable ${l} ; \
      done
```

**Comment by philippjfr** [comment]
> I can't use latest altair-3.0.0, as it needs Jupyterlab-1.

I'm bit confused why altair 3.0 requires an alpha release of JupyterLab, naively that seems like a very strange decision, and I'd be hesitant to let it dictate our release schedule.

**Comment by stonebig** [comment]
https://github.com/altair-viz/altair/issues/1459#issuecomment-487203728

Well, we can use altair-3 I  jupyterlab-0.35.x, but in degraded mode

**Comment by michaelaye** [comment]
@philippjfr There's a technology upgrade in JupyterLab 1.x that altair 3.0 relies on (hence the major version increase). Jake waited apparently 3 months for JLab 1.x to come out but couldn't or didn't want to wait any longer. Note, that altair 3.0 works fine in the classic notebook, this is only an issue for JupyterLab. I also would be interested in having both pyviz and altair available at their newest versions, specifically, because altair 3.0 is a HUGE update with lots of new features.

**Comment by athornton** [comment]
I think we can safely release a "dev" version, with a "next" tag, that will be detected by NPM as JL 1.0.0-alpha compatible, without breaking everyone still using 0.35.x.

Dask-labextension has done this with their 0.4.x-dev releases, so copying what they did is probably the best way to do that.  Ian Rose is probably the best contact, and in general I've found he's very good at explaining things.

**Comment by philippjfr** [comment]
Apologies I've left this sit for so long. I'm happy to release a next tag, I'll make a few suggestions and then let's get this merged and I'll publish to npm.

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@pyviz/jupyterlab_pyviz",
-  "version": "0.7.1",
+  "version": "1.0.0-alpha.6",
```
Comment (philippjfr) [inline package.json]: Our versioning does not mirror the JupyterLab versioning so let's go with:

```suggestion
  "version": "0.8.0-alpha.1",
```

**Comment by philippjfr** [comment]
One small suggestion about the version. Could you also update the version table in the README? Thanks again for the PR!

**Comment by philippjfr** [comment]
@athornton Let me know if you'll get to this soon otherwise I'll probably make a new PR.

**Comment by stonebig** [comment]
@athornton ?

**Comment by athornton** [comment]
Whoops!  Sorry!  Haven't been keeping up with my GH notifications.

Yes, I can do the version changes.

**Comment by philippjfr** [comment]
@stonebig Where is the best place to install JupyterLab 1.0.0-alpha6 from? The tests are currently failing because it's trying to build against older JupyterLab.

**Comment by stonebig** [comment]
I download it from pypi. for example:

````
pip download --dest  C:\WinP\a  --pre `jupyterlab==1.0.0a3
````

**Comment by philippjfr** [comment]
Okay, I'll just merge for now and then try to fix the tests separately.

**Comment by philippjfr** [comment]
Thanks again @athornton!

**Pull request merged**